pkg/container-collection/options.go

```diff
@@ -585,6 +581,34 @@ func WithLinuxNamespaceEnrichment() ContainerCollectionOption {
 	}
 }
 
+// WithOCIConfigEnrichment enriches container using provided OCI config
+func WithOCIConfigEnrichment() ContainerCollectionOption {
```

Comment:
Since the name of this function is not specific to cri-o, should you also enrich from containerd annotations?